boards/arm/frdm_kw41z/Makefile.board

```diff
@@ -2,4 +2,6 @@ DEBUG_SCRIPT = jlink.sh
 
```

Comment:
jlink.sh and pyocd.sh require different OpenSDA firmware, so this would mean you have to update the firmware each time you switch between `make flash` and `make debug`. I think we should pick a preferred firmware for each board, then set the DEBUG_SCRIPT and FLASH_SCRIPT defaults accordingly. In the case of jlink firmware, that means no default FLASH_SCRIPT.